Formula/impute2.rb

```diff
@@ -0,0 +1,25 @@
+class Impute2 < Formula
+  desc "Genotype imputation and haplotype phasing program"
+  homepage "https://mathgen.stats.ox.ac.uk/impute/impute_v2.html"
+
+  # tag "bioinformatics"
+  # doi "10.1371/journal.pgen.1000529"
+
+  version "2.3.2"
+  if OS.mac?
+    url "https://mathgen.stats.ox.ac.uk/impute/impute_v2.3.2_MacOSX_Intel.tgz"
+    sha256 "e91ad1edc66e6174c9dc4fdc06685df7b2d6a9ba4f86477f1a2ceef89a296953"
+  elsif OS.linux?
+    url "https://mathgen.stats.ox.ac.uk/impute/impute_v2.3.2_x86_64_static.tgz"
+    sha256 "95615900464505ddff4251137523740c7f35ac2865eb1eac4d8584ab15a8c45f"
+  end
+
+  def install
+    bin.install "impute2"
+    pkgshare.install "Example"
+  end
+
+  test do
+    assert_match "IMPUTE version", shell_output("#{bin}/impute2 2>&1", 1)
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
This is infeasible I'm afraid, since the program's functionality a) requires a large amount of input data concerning genomic imputation, b) takes a very long time to run.

Reply:
We have some genome/sequence/bioinformatics software with proper tests. One idea could be to have a very small sequence, that runs in short time. If that is not possible at all, then at the very least feed it a too-short sequence and obtain an error?